qdrant-landing/content/documentation/tutorials/async-api.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Asynchronous API
+weight: 18
+---
+
+# Using Qdrant asynchronously
+
+Databases are often launched as separate services and are accessed via a network. All the interactions with them are IO-bound and can 
+be performed asynchronously so as not to waste time actively waiting for a server response. If you use Python, that is achieved by 
+using `async/await` syntax. That lets the interpreter switch to another task while waiting for a response from the server.
+
+Qdrant exposes two interfaces: HTTP and gRPC. The official SDKs for different languages are based on the autogenerated clients, and
+in Python [qdrant-client](https://github.com/qdrant/qdrant-client), you can call all the methods asynchronously. This tutorial presents 
+how to do it.
+
+## When the usage of async API is justified?
+
+If the application you are writing will never support multiple users at once, for example, it is a script run once daily, then there 
+is no need to use async API. But if you are writing a web service that multiple users will use simultaneously, you shouldn't be 
+blocking the threads of the web server as it limits the number of concurrent requests it can handle. In this case, you should use 
+the async API.
+
+Modern web frameworks like [FastAPI](https://fastapi.tiangolo.com/) and [Quart](https://quart.palletsprojects.com/en/latest/) support 
+async API out of the box. Asynchronous code cannot be incorporated into an existing synchronous codebase, so it's better to choose 
+the proper tool from the start.
```

Comment:
This doesn't seem right to me. For example, parts of my code could by synchronus and others could be ansync - it just depends on the task I want that code to do. For example, I could want all my write to disks to be synchronus but any external api calls to be async

Reply:
For sure. But my Python object does not have to exclusively call all async methods. I think async code can be integrated in synchronous code but it has to be done carefully. 

I think my comment is because this statement seems too broad to me. Perhaps it needs to be more specific about the exact anti-pattern. Codebase is a very broad term. Could we say within a method?

Reply:
I changed that part, so it's not that broad anymore.